.github/workflows/deploy-to-k8s.yaml

```diff
@@ -59,6 +59,8 @@ jobs:
         --set email.smtp_password="${{ secrets.SMTP_PASSWORD }}" \
         --set meshweb.backup_s3_base_folder="${{ vars.BACKUP_S3_BASE_FOLDER }}" \
         --set meshweb.backup_s3_bucket_name="${{ secrets.BACKUP_S3_BUCKET_NAME }}" \
+        --set meshweb.join_record_bucket_name="${{ secrets.JOIN_RECORD_BUCKET_NAME }}" \
```

Comment:
Add both to dev3 + gamma envs in UI. `| b64enc` will have a bad day

Reply:
Added. Side note: the dev3 prefix is called `dev3-join-form-submissions`, but we should delete the prefix and make a new one.

Reply:
Thanks